basculehttp/metricListener.go

```diff
@@ -32,13 +32,27 @@ const (
 	defaultServer = "primary"
 )
 
+// MetricListener
```

Comment:
did you mean to write more here?

Reply:
yeah, that's the last file left in need of documentation. 🙂